source/include/ota.h

```diff
@@ -210,13 +210,11 @@ typedef struct OtaJobDocument
  * @ingroup ota_callback_types
  * @brief OTA update complete callback function typedef.
  *
- * The user may register a callback function when initializing the OTA Agent. This
+ * The user has to register a callback function when initializing the OTA Agent. This
```

Comment:
I would prefer "must" to "has to"

Reply:
Fixed in 8ed3d1fabbd5ecd9cd9e602491554bd7c37259b4